specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/netapp.json

```diff
@@ -28,565 +30,599 @@
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/ApiVersionParameter"
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/LocationParameter"
           }
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Azure operation completed successfully.",
             "schema": {
-              "$ref": "#/definitions/regionInfoResource"
+              "$ref": "#/definitions/RegionInfoResource"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "An unexpected error response.",
             "schema": {
               "$ref": "../../../../../common-types/resource-management/v6/types.json#/definitions/ErrorResponse"
             }
           }
         },
         "x-ms-examples": {
           "RegionInfos_Get": {
-            "$ref": "examples/RegionInfos_Get.json"
+            "$ref": "./examples/RegionInfos_Get.json"
           }
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/queryNetworkSiblingSet": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/updateNetworkSiblingSet": {
       "post": {
-        "tags": [
-          "NetApp Resource"
-        ],
-        "summary": "Describe a network sibling set",
-        "description": "Get details of the specified network sibling set.",
-        "operationId": "NetAppResource_QueryNetworkSiblingSet",
+        "operationId": "NetAppResource_UpdateNetworkSiblingSet",
+        "summary": "Update the network features of a network sibling set",
+        "description": "Update the network features of the specified network sibling set.",
         "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
           {
             "name": "body",
             "in": "body",
-            "description": "Network sibling set to query.",
+            "description": "The request body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/QueryNetworkSiblingSetRequest"
-            },
-            "x-ms-client-flatten": true
+              "$ref": "#/definitions/UpdateNetworkSiblingSetRequest"
+            }
```

Comment:
Compared with original swagger https://github.com/Azure/azure-rest-api-specs/blob/fb90eb1bec64c6e8ad3e288a64c84cc18742a394/specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/netapp.json#L572-L590, the `"x-ms-client-flatten": true` is missing. But python is OK for the breaking.